web/Dockerfile

```diff
@@ -1,11 +1,18 @@
-ARG CODE_VERSION=0.6.1
+ARG CODE_VERSION=0.7.0
 FROM kopano/kwebd:${CODE_VERSION}
 
 ARG VCS_REF
 ARG CODE_VERSION
 
 ENV CODE_VERSION="${CODE_VERSION}"
 
+ENV KWEBD_USER root
+ENV KWEBD_GROUP root
+# hadolint ignore=DL3002
+USER root
```

Comment:
This should change to a non root user later on again.

Reply:
without `USER root` kweb does not start since its listening on privileged ports: `2019/08/05 11:24:42 Listen: listen tcp :80: bind: permission denied
`

Reply:
Just change internal ports to 8080 and 8443 should do the trick

Reply:
Ah I remember now - not possible because of internal requests right?

Reply:
yes, that was indeed the reason behind this.

Reply:
So @fbartels to fix this set the caps on the kwebd file

setcap 'cap_net_bind_service=+ep'  /path/to/kwebd

Reply:
Guess that is something worth adding to the standard kwebd docker image